tests/flakyfunctional/xtriggers/00-wall_clock/flow.cylc

```diff
@@ -1,10 +1,10 @@
 #!Jinja2
 
 # Test wall_clock xtrigger: workflow will run to completion and exit if the
-# clock trigger is not satisfied, else stall and abort.
+# clock trigger is not satisfied, else abort on inactivity.
 
 [scheduler]
-    cycle point time zone = '+0530'
+    # Default to time zone Z
```

Comment:
(This test also fails on master, if system time is local in my timezone).

Reply:
I'm working my a home pc, so I can jimmy the system clock to test this.